internal/store/projects.go

```diff
@@ -98,7 +98,8 @@ func (s *Store) GetOrgProjects(orgID string, opts *atlas.ProjectsListOptions) (i
 func (s *Store) Project(id string) (interface{}, error) {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		result, _, err := s.client.(*atlas.Client).Projects.GetOneProject(s.ctx, id)
+		result, res, err := s.clientv2.ProjectsApi.GetProject(s.ctx, id).Execute()
```

Comment:
[nit] It is hard to figure out what execute objects returns. when we have interface and client defined in separate file. the good pattern would be to define result before assigning it:

```
var result apiv2.ResponseObj, res apiv2...
result, res, err = s.clientv2.ProjectsApi
```
It will make our interface approach much better as developers will know exact type instantly.

@matt-condon WDYT

Reply:
This is opinionated. Feel free to ignore. 
One can go to execute methods itself to get the class anyway.